doc/source/spectra.rst

```diff
@@ -58,6 +77,38 @@ Spectrum1DCollection
 This is an object for storing multiple 1D spectra which share the same
 x-axis, e.g. bands in a dispersion plot.
 
+From Spectrum1D
+---------------
+
+If you have multiple ``Spectrum1D`` objects with the same ``x_data``,
+they can be grouped together into a ``Spectrum1DCollection`` object.
+For example:
+
+.. code-block:: py
+
+  from euphonic import Spectrum1D, Spectrum1DCollection
+
+  pdos_si = Spectrum1D.from_json_file('si_pdos.json')
+  pdos_o = Spectrum1D.from_json_file('o_pdos.json')
+
+  pdos_collection = Spectrum1DCollection.from_spectra([pdos_si, pdos_o])
+
+Adding Spectrum1DCollection
+---------------------------
+
+Two ``Spectrum1DCollection`` objects can be added together (provided they
+have the same ``x_data`` axes) with the ``+`` operator. This will append
```

Comment:
I think "concatenate" is more accurate: "append" implies that we mutate one of the objects by adding data from the other.